src/login/logind-button.c

```diff
@@ -63,7 +64,7 @@ void button_free(Button *b) {
                 /* If the device has been unplugged close() returns
                  * ENODEV, let's ignore this, hence we don't use
                  * safe_close() */
-                (void) close(b->fd);
+                (void) asynchronous_close(b->fd);
```

Comment:
This really should have a comment, explaining what is going on here.

Reply:
Since we don't take into account the return value of the close() call we don't have to wait until it return. Is something like : /* The button is freed without waiting for the device to respond at the call */ would be enough ?

Reply:
I think, he asking to put a comment explaining why you are calling `asynchronous_close` and not what this function is doing.

So something like that (but my english is quite bad...):
> The button is freed without waiting for the device to respond, since some devices may take a lot of time to close, this will speed up the startup quite significantly